addons/binding/org.openhab.binding.solaredge/README.md

```diff
@@ -16,16 +16,18 @@ Auto-Discovery is not supported, as access requires authentication.
 The following configuration parameters are available for this thing:
 
 - **tokenOrApiKey** (required)
-Either the official API Key for using the public API or when using the inofficial private API: a token which can be retrieved from browser's cookie store when logged into the solaredge website. It is called 'SPRING_SECURITY_REMEMBER_ME_COOKIE'
+Either the [official API Key](https://www.youtube.com/watch?v=iR26nmL5bXg) for using the public API or when using the inofficial private API: a token which can be retrieved from browser's cookie store when logged into the solaredge website. It is called 'SPRING_SECURITY_REMEMBER_ME_COOKIE' When using this token, see also usePrivateApi and meterInstalled.
+
+For Firefox, install an addon such as [Cookie Manager](https://addons.mozilla.org/firefox/addon/a-cookie-manager) to retrieve the token, as this browser no longer offers viewing of cookies with its standard functionality.
```

Comment:
Do you really need to install an add-on for this? Can we use the [Storage Inspector](https://developer.mozilla.org/en-US/docs/Tools/Storage_Inspector) of the built-in [Developer Tools](https://developer.mozilla.org/en-US/docs/Tools)?